packages/expo/src/ImagePicker/MediaTypeOptions.ts

```diff
@@ -0,0 +1,5 @@
+export default {
```

Comment:
Good opportunity to use an enum here instead:
```suggestion
export default enum MediaTypeOptions {
```